conans/model/version_range.py

```diff
@@ -94,6 +94,11 @@ def __init__(self, expression):
         prereleases = False
         for t in tokens[1:]:
             if "include_prerelease" in t:
+                if "include_prerelease=" in t:
+                    from conan.api.output import ConanOutput
+                    ConanOutput().warning(
+                        f'include_prerelease version range option in "{expression}" does not take an attribute, '
+                        'its presence unconditionally enables prereleases')
```

Comment:
Having a simple unittest checking that this message is appearing could be great: `conans/test/unittests/model/version/test_version_range.py`